src/components/messenger/list/conversation-item/index.tsx

```diff
@@ -94,17 +111,20 @@ export class ConversationItem extends React.Component<Properties> {
       />
     );
   }
+
   renderMoreMenu() {
     const stopPropagation = (e) => {
       e.stopPropagation();
     };
 
     return (
-      <div {...cn('more-menu-container')} onClick={stopPropagation}>
+      <div onClick={stopPropagation}>
```

Comment:
Does this still need the `stopPropogation`?

Reply:
Yeah I tested with/without and `stopPropogation` is still required to prevent the conversation being selected (opened)

Reply:
the conversation opens when you right click? I thought this was just to prevent it from opening when you click the 3 dots but since we don't have the 3 dots now.... it shouldn't matter?

Reply:
Not if you right click, but when selecting the menu item `Favorite` or `Unfavorite`, the conversation will be selected and open (without the `stopPropogation`)

Reply:
That feels more like a bug in the DropdownMenu then.

Reply:
Do you think with the current implementation this will be safe enough to remove favorites feature flag and deploy to prod? I'm assuming I shouldn't wait for a QA on this

Reply:
I think it's good enough to deploy. Whether it should be enabled is Kelly's decision.